sematic/api_client.py

```diff
@@ -90,17 +91,16 @@ def _get_artifact(artifact_id: str) -> Artifact:
 
 @retry(tries=3, delay=10, jitter=1)
 def store_artifact_bytes(artifact_id: str, bytes_: bytes) -> None:
-    response = _get(f"/artifacts/{artifact_id}/location")
+    response = _get(f"/uploads/artifacts/{artifact_id}/location")
 
     location: str = response["location"]
+    headers: Dict[str, str] = response["headers"]
 
-    put = requests.put if location.startswith("https://") else _put
-
-    put(location, data=bytes_)
+    requests.put(location, data=bytes_, headers=headers)
 
 
 def _get_artifact_bytes(artifact_id: str) -> bytes:
-    return _get(f"/artifacts/{artifact_id}/data", decode_json=False)
+    return _get(f"/uploads/artifacts/{artifact_id}/data", decode_json=False)
```

Comment:
These values are uploaded using this endpoint, but then they are also downloaded, from these, such as in this example. Instead of suggesting only the "upload" direction via the endpoint name, how about having the endpoint be "storage" instead?

PUT to / GET from "uploads/artifacts" vs PUT to / GET from "storage/artifacts".

The `download_endpoint` function registered to serve the `/api/v1/uploads` endpoint in `local_storage.py` is further evidence for the need of this renaming.